src/browser/modules/EditorInfo/EntityType.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {
+  DrawerSubHeader,
+  DrawerSection,
+  DrawerSectionBody
+} from 'browser-components/drawer'
+
+export const EntityType = ({ itemType = '' }) => {
```

Comment:
write up JSDoc for this functional component

Reply:
Done with JSDoc creation